pipelines/datapackage/run.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# [wf] execute run.sh stage
+
+# [wf] data folder
+mkdir data
+
+# [wf] Installing dependencies
+if python -c "import wget" &> /dev/null; then
+    echo 'requirements already installed.'
+else
+    echo 'Installing dependencies'
+    pip install wget
+fi
+
+# [wf] running script generate-data
+python scripts/generate-data.py
```

Comment:
not a big deal, but sometimes it's nice to have a newline at the end of the file (e.g. to play nice with a terminal if someone does `cat run.sh`)